modules/data.atmosphere/R/ERA5_download.R

```diff
@@ -0,0 +1,142 @@
+#' @description
+#' This function helps to download the yearly ERA5 data based on the prescribed features using the CDS API.
+#' @title ERA5_cds_annual_download
+#' 
+#' @param years Numeric: a series of years to be downloaded (e.g., 2012:2021).
+#' @param months List: a list contains months to be downloaded (e.g., list("01", "02") to download files in Jan and Feb).
+#' @param days List: a list contains days to be downloaded (e.g., list("01", "02") to download files in the first and second days).
+#' @param times List: a list contains times to be downloaded (e.g., list('00:00','03:00') to download files for the times 12:00 and 3:00 am UTC).
+#' @param area Character: a string contains the bounding box (formatted as "North/West/South/East") to be downloaded (e.g., "85/-179/7/-20").
+#' @param variables List: a list contains variables to be downloaded (e.g., list("2m_temperature","surface_pressure")).
+#' @param outdir Character: physical path where the ERA5 data are stored.
+#' @param auto.create.key Boolean: decide if we want to generate the CDS RC file if it doesn't exist, the default is TRUE.
+#'
+#' @return A vector containing file paths to the downloaded files.
+#' @export
+#' 
+#' @examples
+#' @author Dongchen Zhang
+ERA5_cds_annual_download <- function(years, months, days, times, area, variables, outdir, auto.create.key = T) {
+  options(timeout=360000)
```

Comment:
I'd make this an optional argument to the function with a default, rather than hard coding.

Reply:
Fixed.